src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const treeHeaders = document.querySelectorAll('.tree > li > :first-child');
```

Comment:
Ensure that the selector '.tree > li > :first-child' correctly targets the elements you intend to modify. If there are any changes in the HTML structure, this selector might need adjustments.